### ISSUE TITLE
Steam/Server related issue Unable to update in game name

### ISSUE THREAD

**Issue by Edgar-BlipparDev** [StrangeLoopGames/EcoIssues#6113]
Seems there is issue where people with special characters in name unable to vote...

I tried changing name while it updates on my steam the name in game does not update presume some sort of sync issue...

**Comment by MarjieAVolk** [comment]
Yeah you can't change your username on an existing server right now.  If you join a new server, it will take your new name.

**Comment by Edgar-BlipparDev** [comment]
That is for sure issue I had couple people to join with offensive names and I asked them to change it even thought they complied they were unable to do anything about it.. It makes it difficult if you have server with kids

**Comment by MarjieAVolk** [comment]
Do you know which character(s) didn't work in this case?

**Comment by MarjieAVolk** [comment]
Well I guess this bug is where we are tracking special characters + laws: https://github.com/StrangeLoopGames/EcoIssues/issues/4242  I will look into voting when I work on that too.

So I guess I will leave this bug open as the request for changing names.

**Comment by Edgar-BlipparDev** [comment]
I have user with NoobDickBob and he is unable to change name including me I dont know how steam and game sync is handled but that would be my first guess... I presume worlds just record name upon first time only

**Comment by Thyriel** [comment]
Maybe just add an admincommand to rename players ? Would prevent abusing a rename too

**Comment by Edgar-BlipparDev** [comment]
That would be handy

**Comment by MarjieAVolk** [comment]
There is a technical reason that is preventing us from allowing name changes unfortunately, it wasn't just done to prevent abuse.  We can do it, it will just be a bigger task.

**Comment by Edgar-BlipparDev** [comment]
Hmm I imagine name shouldn't matter so much anyway since all client have their unique steamId It would be just overwriting string names but I guess you guys might have some back end madness to work through either way thanks for taking notice on this
